src/Tools/dotnet-monitor/CollectionRules/Options/Triggers/SystemDiagnosticsMetricsOptions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.CollectionRuleDefaultsInterfaces;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Triggers
+{
+    /// <summary>
+    /// Options for the SystemDiagnosticsMetrics trigger.
+    /// </summary>
+    internal sealed partial class SystemDiagnosticsMetricsOptions : ISlidingWindowDurationProperties
+    {
+        public string ProviderName { get; set; }
```

Comment:
What is the purpose of ProviderName and CounterName here?

Reply:
https://github.com/dotnet/dotnet-monitor/pull/3812#discussion_r1122272607